client/components/product-list-item.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import priceFormatter from '../price-format';
+
+export default function ProductListItem(props) {
+  const { image, name, price, shortDescription } = props.productInfo;
+
+  return (
+    <div className="card col-md-4 mb-3">
+      <img src={image} className="card-img-top product-image" alt="A product image"/>
+      <div className="card-body">
+        <h5 className="card-title">{name}</h5>
+        <p className="card-text">{priceFormatter(price)}</p>
+        <p className="card-text">{shortDescription}</p>
+      </div>
+    </div>
```

Comment:
Instead of making each cart a `col`, wrap each card _in_ a `col`. That should fix your horizontal margin issue.